website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -269,6 +269,8 @@ An `oms_agent` block exports the following:
 
 * `log_analytics_workspace_id` - The ID of the Log Analytics Workspace to which the OMS Agent should send data.
 
+* `msi_auth_for_monitoring_enabled` - Is managed identity authentication for monitoring enabled?
```

Comment:
This property hasn't been added to the Kubernetes data source yet, so could you either remove this or add it to the data source?